### ISSUE TITLE
Error: temporary source directory must have been generated by hardhat-dependency-compiler

### ISSUE THREAD

**Issue by 134dd3v** [ItsNickBarry/hardhat-dependency-compiler#9]
Steps to reproduce:
- Interrupt during compilation (Cmd + C)
- Delete the files which are untracked in git
- Try to compile again. Compilation gets locked until this is fixed.

Some hint how to reset this in the error msg would save some time. 

To get rid of this error: There is empty directory `hardhat-dependency-compiler` in the contracts folder. And just have to delete that.

**Comment by ItsNickBarry** [comment]
Are you able to reproduce consistently?  The error is thrown if the output directory exists and does not contain a special plugin-generated tracker file, which is used to indicate that the directory was also plugin-generated and can be safely cleared.

The tracker is generated immediately after the directory:
```javascript
fs.mkdirSync(directory);
fs.writeFileSync(tracker, `directory approved for write access by ${ name }\n`);
```

So, for the directory to be generated without the tracker file, compilation would have to be interrupted between those two calls.  How often is this happening?

**Comment by ItsNickBarry** [comment]
Should be fixed in `1.1.3`, though I'm still curious about how this could be consistently reproduced.

**Comment by 134dd3v** [comment]
> So, for the directory to be generated without the tracker file

It did generate the tracker file as expected. I deleted the tracker file (I thought it won't matter). I was compiling the contracts which does take a while, then I had to interrupt it to get to something important on another branch, and hence had to delete the tracker files (but the directory doesn't get deleted when done with VSCode's Source Control UI). This is something one would rarely do but I just ran into this situation in which I was wondering what do I need to do in order to make it work lol and I was actually in a hurry.
